adafruit_platformdetect/board.py

```diff
@@ -60,6 +60,9 @@
 
 FTDI_FT232H                 = "FT232H"
 DRAGONBOARD_410C            = "DRAGONBOARD_410C"
+
+SIFIVEHFUA00                = "SIFIVEHFUA00"

```

Comment:
We're trying to decouple the board from the chip, so maybe more like the board name such as SIFIVE_UNLEASHED or something like that.